portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,35 +24,81 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that handles comment data. */
```

Comment:
nit: "handles" is a somewhat ambiguous definition. Does this Servlet receive persisted data? Does it only handle the Http Request and delegate the response somewhere else? Etc

Reply:
I've rewritten this!